src/vs/workbench/contrib/debug/common/replModel.ts

```diff
@@ -119,12 +120,12 @@ export class ReplModel {
 		this.addReplElement(result);
 	}
 
-	appendToRepl(data: string | IExpression, sev: severity, source?: IReplElementSource): void {
+	appendToRepl(session: IDebugSession | undefined, data: string | IExpression, sev: severity, source?: IReplElementSource): void {
```

Comment:
Here the session should always be defined, it should never be undefined (I think that case can not happen)